rskj-core/src/test/java/co/rsk/peg/BridgeSupportRegisterBtcTransactionTest.java

```diff
@@ -2401,10 +2402,7 @@ void flyover_segwit_as_migration_utxo(
 
         FederationTestUtils.addSignatures(retiringFederation, retiringFedSigners, btcTransaction);
 
-        TransactionWitness txWitness = new TransactionWitness(1);
-        txWitness.setPush(0, new byte[72]); // push for signatures
-        txWitness.setPush(1, retiringFederation.getRedeemScript().getProgram());
-        btcTransaction.setWitness(0, txWitness);
+        addWitnessToBtcTxInputWithEmptySignatures(btcTransaction, FIRST_INPUT_INDEX, retiringFederation.getRedeemScript());
```

Comment:
```suggestion
        addBaseWitnessToBtcTxInputThatSpendsFromRedeemScript(btcTransaction, FIRST_INPUT_INDEX, retiringFederation.getRedeemScript());
```
wdyt? A bit more descriptive (i think saying _base_ witness implies you are adding the rs but no signatures)

Reply:
agree! done

Reply:
I think `addBaseWitnessToBtcTxInput` should be enough. I don't see the need to add this part "ThatSpendsFromRedeemScript".

Reply:
I will change it for "create..." because it doesn't add any redeemScripts but builds it